tasks/test.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (grunt) {
       },
       src: [
         'test/**/*.js',
-        '!test/lib/**'
+        '!test/util/**'
```

Comment:
Looks like you need to revert this after switching back to from `util` to `lib`?